cedar-drt/fuzz/src/dump.rs

```diff
@@ -135,6 +131,18 @@ pub fn dump<'a>(
     Ok(())
 }
 
+/// Check whether a policyset passes validation
+fn passes_validation(schema: &SchemaFragment, policies: &PolicySet) -> bool {
+    if let Ok(schema) = ValidatorSchema::try_from(schema.clone()) {
```

Comment:
nit: since we unconditionally clone `schema`, it might make sense to take an owned `SchemaFragment` in this function, allowing caller to give us ownership if they have it.  (This won't make a difference in this particular case, so I guess it's a style judgment call.)